### PR TITLE
Turbine-Ruby CLI Stub 

### PR DESCRIPTION
**Pull request by anna-cross**
## Description of change

Stubs for turbine-ruby cli. Contains functions needed by interfaces 

https://github.com/meroxa/cli/blob/a3f45174615af714697d5cf043dd8751d3bbb831/cmd/meroxa/turbine/interface.go#L7

Fixes https://github.com/meroxa/cli/issues/521

## Type of change

- [x]  New feature (stub)

## How was this tested?

- [x]  Unit Tests

### CODE REVIEW

File: cmd/meroxa/turbine/ruby/run.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import (
+	"context"
+)
+
+const GrpcServerPort = 50500
+
+func (t *turbineRbCLI) Run(ctx context.Context) (err error) {
+	// Run turbine-core gRPC server
+	return err
```
Comment (dianadoherty) [inline cmd/meroxa/turbine/ruby/run.go:11]: this will return nil, so I recommend adding an error

File: cmd/meroxa/turbine/ruby/build.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package turbinerb
+
+import (
+	"context"
+)
+
+func (t *turbineRbCLI) Build(ctx context.Context, appName string, platform bool) (string, error) {
+	return "", nil
```
Comment (dianadoherty) [inline cmd/meroxa/turbine/ruby/build.go:8]: I think these should return a "not implemented" error

Comment (anna-cross) [reply cmd/meroxa/turbine/ruby/build.go]: Sounds good!

File: cmd/meroxa/turbine/ruby/run.go (hunk 1 above)
Comment (anna-cross) [reply cmd/meroxa/turbine/ruby/run.go]: Sounds good!

**Review by dianadoherty — APPROVED** [review]

**Review by raulb — APPROVED** [review]
@dianadoherty do you think we should a feature flag for the ruby path until we finish it?

**Comment by dianadoherty** [comment]
> @dianadoherty do you think we should a feature flag for the ruby path until we finish it?


because turbine ruby commands are not exposed in this PR, this PR can be merged. But we should definitely add a flag for any future pr

**Pull request merged**